scripts/check_makefile.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+repo="$1"
```

Comment:
I'm a bit freaked out with bash variable scopes, and fear the variables in this script may interfere with others from other scripts, so I would suggest to enclose all substantial code inside functions, with `local` variables only and proper `return` / `echo` for values to be shared between scripts.

Reply:
I've changed the way scripts work, now all functions and scripts work in their subshells so no possible variable interference